src/MatrixExp_Hamiltonians.jl

```diff
@@ -318,84 +318,52 @@ function propagator_linear_inversion_pulse(ω1, T, B1, R2s, _, dR2sdB1, grad_typ
 end
 
 function z_rotation_propagator(rfphase_increment, _)
-    u_rot = @SMatrix [cos(rfphase_increment) -sin(rfphase_increment) 0 0 0 0;
-                      sin(rfphase_increment)  cos(rfphase_increment) 0 0 0 0;
-                                     0                0  1 0 0 0;
-                                     0                0  0 1 0 0;
-                                     0                0  0 0 1 0;
-                                     0                0  0 0 0 1]
+	(sϕ,cϕ) = sincos(rfphase_increment)
+    u_rot = @SMatrix [cϕ -sϕ 0 0 0 0;
+                      sϕ  cϕ 0 0 0 0;
+                       0   0 1 0 0 0;
+                       0   0 0 1 0 0;
+                       0   0 0 0 1 0;
+                       0   0 0 0 0 1]
     return u_rot
 end
 
-function z_rotation_propagator(rfphase_increment, grad::grad_param)
-    u_rot = @SMatrix [cos(rfphase_increment) -sin(rfphase_increment) 0 0 0 0                 0                0 0 0 0;
-                      sin(rfphase_increment)  cos(rfphase_increment) 0 0 0 0                 0                0 0 0 0;
-                      0                 0                1 0 0 0                 0                0 0 0 0;
-                      0                 0                0 1 0 0                 0                0 0 0 0;
-                      0                 0                0 0 1 0                 0                0 0 0 0;
-                      0                 0                0 0 0 cos(rfphase_increment) -sin(rfphase_increment) 0 0 0 0;
-                      0                 0                0 0 0 sin(rfphase_increment)  cos(rfphase_increment) 0 0 0 0;
-                      0                 0                0 0 0 0                 0                1 0 0 0;
-                      0                 0                0 0 0 0                 0                0 1 0 0;
-                      0                 0                0 0 0 0                 0                0 0 1 0;
-                      0                 0                0 0 0 0                 0                0 0 0 1]
+function z_rotation_propagator(rfphase_increment, ::grad_param)
+	(sϕ,cϕ) = sincos(rfphase_increment)
```

Comment:
Do you mind removing the brackets and adding a whitespace `sϕ, cϕ = sincos(rfphase_increment)`, and aligning the line with other code?